bin/bs_ios

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# bs_ios uploads app binaries for UI testing on BrowserStack.
+#
+# This version of the script works only with the V2 endpoint (XC Test Plans).
+# This script assumes that Xcode Test Plan name is TestPlan.xctestplan.
+#
+# It forwards all arguments to "patrol build ios", so you can pass --target,
+# --flavor, --exclude etc. just as you would pass them to "patrol_cli".
+#
+# It can also be configured with the following environment variables:
+BROWSERSTACK_CREDS="${BROWSERSTACK_CREDS:-}"
+BROWSERSTACK_PROJECT="${BROWSERSTACK_PROJECT:-}"
+BROWSERSTACK_IOS_DEVICES="${BROWSERSTACK_IOS_DEVICES:-}"
+BROWSERSTACK_SKIP_BUILD="${BROWSERSTACK_SKIP_BUILD:-0}"
```

Comment:
What do you mean? Like, it should be a flag: `--skip-build`?

Reply:
I'm against it since it:
1. won't be consistent with other options
2. will increase complexity of this script

+ setting env var is not that hard, come on

Reply:
but if you set this today you won't remember that it has been set up yesterday

Reply:
> but if you set this today you won't remember that it has been set up yesterday

You can set it in the env var in a script [like explained here](https://github.com/leancodepl/mobile-tools/pull/51#discussion_r1409182823). BTW the script also prints info if this env var is disabled.

And you can run it like: `BROWSERSTACK_SKIP_BUILD=1 bs_ios`